integration-libs/cdc/manage-users/manage-users.service.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { CdcJsService } from '@spartacus/cdc/root';
+import {
+  GlobalMessageService,
+  GlobalMessageType,
+  WindowRef,
+} from '@spartacus/core';
+@Injectable({
+  providedIn: 'root',
+})
+export class manageUsersService {
```

Comment:
change the class name to `ManageUsersService`

Reply:
done